Projects/count-down/main.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+# The countdown function is defined below
+
+
+def countdown(t):
+
+    while t:
+
+        mins, secs = divmod(t, 60)
+
+        timer = '{:02d}:{:02d}'.format(mins, secs)
+
+        print(timer, end="\r")
+
+        time.sleep(1)
+
+        t -= 1
+
+    print("Ring Ring, it's time!")
+
+# Ask the user to enter the countdown period in seconds
+
+
+t = int(input("Enter the time in seconds: "))
+
+# function call
+
+countdown(t)
```

Comment:
Why is the count-down timer code here when the specified Issue and PR are for Tic-Tac-Toe?

Reply:
I'm sorry for that. Please review my project now.